canonical-web-and-design/jp.ubuntu.com
# Remove the use of the used_categories on the blog card

## Done
Remove the use of the used_categories on the blog card

## QA
- Open the demo
- Go to /blog
- Check it renders and there are no errors
- Check live 500's

## Issue / Card
Fixes https://github.com/canonical-web-and-design/jp.ubuntu.com/issues/320
